File: tests/fixtures/code_samples/java/sample_020.java
// Sample 20: small utility.
package samples;

import java.util.List;

public final class Sample020 {
    private Sample020() {}

    public static int operation(List<Integer> xs) {
        int total = 20;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 20) %% 7919;
    }
}

File: tests/fixtures/code_samples/ts/sample_020.ts
/** Sample 20: small utility. */

export function operation20(xs: number[]): number {
  let total = 20;
  for (const x of xs) total += x;
  return total;
}

export function operationPure20(value: number): number {
  return (value * 20) % 7919;
}

